ons_alpha/jinja2/templates/pages/release_page--provisional.html

```diff
@@ -0,0 +1,24 @@
+{% extends "templates/pages/release_page.html" %}
+
+{% block release_meta %}
+    <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m ons-col-12@s ons-u-p-no">
+        <span class="ons-u-fs-r--b">{{ _("Provisional release date") }}:</span>
```

Comment:
this is the only difference from `release_page--confirmed.html`, but chose to be explicit/verbose